server/images.go

```diff
@@ -62,7 +62,7 @@ type Model struct {
 	Template *template.Template
 }
 
-func (m *Model) Has(caps ...Capability) bool {
+func (m *Model) Has(caps ...Capability) (missing []Capability) {
```

Comment:
This felt clearer when it returned `bool`

Reply:
this should communicate what is missing which a bool return value can't do

Reply:
Yes but it could be error prone (see below) – could we have the caller check in a loop?